components/automate-chef-io/content/docs/api-tokens.md

```diff
@@ -121,7 +125,7 @@ compliance resource.
 
 3. Copy the token returned.
 
-4. Create policies to permit that client to read `compliance:*`. For more information, see [policies]({{< relref "authorization-overview.md" >}}).
+4. Create policies to permit that client to read `compliance:*`. For more information, see [IAM v1 overview]({{< relref "iam-v1-overview.md" >}}).
```

Comment:
maybe capitalize "Overview"

Reply:
gonna wait for @mjingle on this one, and then i can update them all if she wants that change

Reply:
I would update this to "Overview," which is also consistent with the other instances of IAM v1 Overview in this PR. :)